lib/mongo-stream.js

```diff
@@ -25,8 +27,20 @@ MongoStream.prototype._write = function (chunk, encoding, done) {
 
     dao.writeLog(data, function (err) {
       if (err) {
-        console.error('fh-bunyan failed to write log to mongodb', err);
-        console.error((err && err.stack) ? err.stack : 'no stack available');
+        if (isMongoConnectionError(err)) {                    //if the error is caused by the unavailability of mongodb
+          if (!flag) {
```

Comment:
@evanshortiss  Nooooo. That's is actually how he originally had it :) The problem is that we need 3 separate code branches of code to cover the possibilities:

1) Not a mongo connection error, log it
2) Mongo connection error, and we haven't seen one in a minute, so log it
3) Mongo connection error, but we have already logged one in the past minute, so ignore it.